Tests/Marketplace/download_private_id_set.py

```diff
@@ -0,0 +1,98 @@
+import os
+import json
+import argparse
+from Tests.Marketplace.marketplace_services import init_storage_client
+
+
+STORAGE_ID_SET_PATH = 'content/private_id_set.json'
+ARTIFACTS_PATH = '/home/runner/work/content-private/content-private/content/artifacts'
+
+
+def create_empty_id_set_in_artifacts(private_id_set_artifacts_path):
+    empty_id_set = {
+        "scripts": [],
+        "playbooks": [],
+        "integrations": [],
+        "TestPlaybooks": [],
+        "Classifiers": [],
+        "Dashboards": [],
+        "IncidentFields": [],
+        "IncidentTypes": [],
+        "IndicatorFields": [],
+        "IndicatorTypes": [],
+        "Layouts": [],
+        "Reports": [],
+        "Widgets": [],
+        "Mappers": []
+    }
+    with open(private_id_set_artifacts_path, 'w') as id_set:
+        json.dump(empty_id_set, id_set)
+
+
+def file_exists_in_bucket(public_storage_bucket):
```

Comment:
what file? let the name of the function reflect that

Reply:
done